doc/classes/PhysicsServer2D.xml

```diff
@@ -592,11 +592,12 @@
 			<param index="1" name="callable" type="Callable" />
 			<param index="2" name="userdata" type="Variant" default="null" />
 			<description>
-				Sets the function used to calculate physics for the body, if that body allows it (see [method body_set_omit_force_integration]).
-				The force integration function takes the following two parameters:
+				Sets the body's force integration callback function, which can be used e.g. to update the body's linear and angular velocity (for example, based on contact with other bodies).
+				If [param userdata] is not [code]null[/code], the function [param callable] must take the following two parameters:
 				1. a [PhysicsDirectBodyState2D] [code]state[/code]: used to retrieve and modify the body's state,
-				2. a [Variant] [param userdata]: optional user data.
-				[b]Note:[/b] This callback is currently not called in Godot Physics.
+				2. a [Variant] [code skip-lint]userdata[/code]: the user data [param userdata].
+				If [param userdata] is [code]null[/code], then [param callable] must take only the [code]state[/code] parameter.
+				[b]Note:[/b] The function [param callable] will be called every physics tick, and it will be called before the standard force integration if the latter is enabled (see [method body_set_omit_force_integration]).
```

Comment:
```suggestion
				2. A [Variant] [code skip-lint]userdata[/code], which is the same as [param userdata].
				If [param userdata] is [code]null[/code], [param callable] must take only the [code]state[/code] parameter.
				[b]Note:[/b] The function [param callable] will be called every physics tick, and it will be called before the standard force integration if enabled (see [method body_set_omit_force_integration]).
```
I think the last note is so "noteworthy" it shouldn't be a note. Should probably be mentioned way sooner what this "force integration callback" is used for.